calibration/CTDMOG/CGINS-CTDMOG-13621__20170404.csv

```diff
@@ -20,6 +20,6 @@ serial,name,value,notes
 37-13621,CC_ptcb1,-3.500000e-004,
 37-13621,CC_ptcb2,0.00E+00,
 37-13621,CC_ptempa0,1.564118e+002,
-37-13621,CC_ptempa1,-7.167185e-002,
-37-13621,CC_ptempa2,9.427969e-007,
-37-13621,CC_wbotc,3.8230e-007,
+37-13621,CC_ptempa1,-7.167184e-02,
+37-13621,CC_ptempa2,9.427968e-07,
+37-13621,CC_wbotc,3.822987e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT